src/features/list-view/examples/ListViewExamples.js

```diff
@@ -7,21 +7,21 @@ import ListView from '../ListView';
 
 const generateRandomString = value => String.fromCharCode((value % 65) + 65);
 
-const getGridHeader = columnIndex => generateRandomString(columnIndex);
+const getGridHeader = columnIndex => `Header ${generateRandomString(columnIndex)}`;
 
-const getGridCell = ({ columnIndex, rowIndex }) => generateRandomString(columnIndex + rowIndex);
+const getGridCell = ({ columnIndex, rowIndex }) => `Row ${rowIndex}, Column ${columnIndex}`;
 
 const ListViewExamples = () => {
     return (
-        <div style={{ height: '1000px' }}>
+        <div style={{ height: '300px' }}>
```

Comment:
Could this height be moved to a constant?

Reply:
I'm less concerned given that it's a README.

Do you see any functional or readability issues with the change-set?